new-site/src/components/Hero.scss

```diff
@@ -0,0 +1,149 @@
+@import "./utils.scss";
+
+.hero-container {
+  --koros-height: 145px;
+  --hero-height: 540px;
+  --hero-small-left: calc((100vw - 100%) / 2);
+  --hero-large-content-left: calc((100vw + var(--spacing-s) - var(--container-width-xl)) / 2);
+
+  background-color: var(--color-coat-of-arms);
+  box-sizing: border-box;
+  position: relative;
+  left: calc(var(--hero-small-left) * -1);
+  width: 100vw;
+
+  @include x-large-and-above-screen {
+    left: calc(var(--hero-large-content-left) * -1);
+    padding-left: var(--spacing-s);
+    z-index: 1;
+    width: 100vw;
+  }
+}
+
+.hero-bg-image {
+  aspect-ratio: 16 / 9;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  margin-top: calc(var(--koros-height) / -2);
+
+  @include x-large-and-above-screen {
+    aspect-ratio: unset;
+    background-size: cover;
+    background-position: center;
+    height: 100%;
+    margin-top: 0;
+    left: var(--hero-large-content-left);
+    right: 0;
+    position: absolute;
+    top: 0;
+  }
+}
+
+.hero-wrapper {
+  margin: 0 auto;
+  max-width: var(--container-width-xl);
+}
+
+.hero {
+  color: var(--color-white);
+  margin: 0 auto;
+  overflow: hidden;
+  position: relative;
+  width: 100%;
+
+  @include x-large-and-above-screen {
+    min-height: var(--hero-height);
+  }
+}
+
+.hero-content {
+  --triangle-top-corner-x: calc(100% + var(--spacing-layout-xl));
+  --triangle-bottom-corner-x: calc(100% + var(--koros-height) + var(--spacing-layout-xl));
+
+  background-color: var(--color-coat-of-arms);
+  box-sizing: border-box;
+  left: var(--hero-small-left);
+  padding: var(--spacing-s) var(--spacing-s);
+  position: relative;
+  z-index: 3;
+
+  @include small-medium-and-above-screen {
+    padding: var(--spacing-s) var(--spacing-m);
+  }
+
+  @include x-large-and-above-screen {
+    clip-path: polygon(0 0, var(--triangle-top-corner-x) 0, 0 var(--triangle-bottom-corner-x), 0% 50%);
+    height: 100%;
+    left: auto;
+    padding: var(--spacing-layout-m) 0 var(--spacing-layout-m) 0;
+    width: 67%;
+
+    &:after {
+      clear: both;
+      content: "";
+      display: table;
+    }
+  }
+}
+
+.hero-content-text {
+  height: 100%;
+  width: 100%;
```

Comment:
I think these height and width might be by default 100%, since I noticed no change when I omitted them?

Reply:
- [ ] Check if height and width can be omitted